content/applications/finance/payment_acquirers/flutterwave.rst

```diff
@@ -0,0 +1,39 @@
+===========
+Flutterwave
+===========
+
+`Flutterwave <https://flutterwave.com/>`_ is an online payments provider established in Nigeria and
+covering several African countries and payment methods.
+
+.. _payment_acquirers/flutterwave/configure_dashboard:
+
+Configuration on Flutterwave Dashboard
+======================================
+
+#. Log into `Flutterwave Dashboard <https://dashboard.flutterwave.com/>`_ and go to
+   :menuselection:`Settings --> API`. Copy the values of the :guilabel:`Public Key` and
+   :guilabel:`Secret Key` fields and save them for later.
+#. | Go to :menuselection:`Settings --> Webhooks` and enter your Odoo database URL followed by
+     `/payment/flutterwave/webhook` in the :guilabel:`URL` text field.
+   | For example: `https://yourcompany.odoo.com/payment/flutterwave/webhook`.
+#. Fill the :guilabel:`Secret hash` with a password that you generate and save its value for later.
+#. Make sure all the remaining checkboxes are ticked.
+#. Click on **Save** to finalize the configuration.
+
+.. _payment_acquirers/flutterwave/configure_odoo:
+
+Configuration on Odoo
+=====================
+
+#. :ref:`Navigate to the payment acquirer Flutterwave and enable it <payment_acquirers/add_new>`.
```

Comment:
:ref:`Navigate to the payment acquirer Flutterwave <payment_acquirers/add_new>` and change its state to Enabled.